apps/web/components/team/screens/Team.tsx

```diff
@@ -2,6 +2,7 @@ import Link from "next/link";
 
 import { WEBAPP_URL } from "@calcom/lib/constants";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { md } from "@calcom/lib/markdownIt";
```

Comment:
I think it would make sense to do this on the server side, as a prisma middleware? We already do this for the event types as well (`packages/prisma/middleware/eventTypeDescriptionParseAndSanitize.ts`). Although I did see that sometimes we tend to do this in other places as well. Keeping it in the prisma middleware makes the most sense I think.